geometry_msgs/Pose[] poses

---

bool response